src/components/menu-bar/menu-bar.jsx

```diff
@@ -468,17 +469,26 @@ class MenuBar extends React.Component {
                         <FormattedMessage {...ariaMessages.tutorials} />
                     </div>
                     <Divider className={classNames(styles.divider)} />
-                    <div className={classNames(styles.menuBarItem, styles.growable)}>
-                        <MenuBarItemTooltip
-                            enable
-                            id="title-field"
-                        >
-                            <ProjectTitleInput
-                                className={classNames(styles.titleFieldGrowable)}
-                                onUpdateProjectTitle={this.props.onUpdateProjectTitle}
-                            />
-                        </MenuBarItemTooltip>
-                    </div>
+                    {(this.props.authorUsername && this.props.authorUsername !== this.props.username) ? (
```

Comment:
It seems like the toggle for this type of display would be whether you should be able to edit the title or not, since in addition to showing the author information, it changes the title field from an editable one to static output.  I'm not seeing any capability flag that already represents that though.

Reply:
good point, i felt a little awkwardness here with GUI having the responsibility to interpret that it _should_ show author info instead of title. Maybe I should create a `canEditTitle` flag from www?

Reply:
That makes sense. We still want to be able edit the title in the beta — is that accounted for here?

Reply:
In my current code, the title stays editable in the beta because authorUsername is not set. But if I introduce a canEditTitle flag, I'd have www set it, and the playground code set it too.